github_connector/views/view_github_team_partner.xml

```diff
@@ -21,11 +21,11 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
             <tree decoration-warning="role == 'member'">
                 <field
                     name="team_id"
-                    invisible="context.get('search_default_team_id')"
+                    attrs="{'invisible': 'context.get(\'search_default_team_id\')'}"
```

Comment:
Incorrect expression:
```suggestion
                    attrs="{'invisible': context.get('search_default_team_id')}"
```
and check the rest.

Reply:
If I do that, I get this error:

```
Odoo Server Error

Traceback (most recent call last):
  File "/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_http.py", line 237, in _dispatch
    result = request.dispatch()
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 683, in dispatch
    result = self._call_function(**self.params)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 359, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 347, in checked_call
    result = self.endpoint(*a, **kw)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 912, in __call__
    return self.method(*args, **kw)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 531, in response_wrap
    response = f(*args, **kw)
  File "/opt/odoo/auto/addons/web/controllers/main.py", line 1389, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/opt/odoo/auto/addons/web/controllers/main.py", line 1381, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/api.py", line 392, in call_kw
    result = _call_kw_model(method, model, args, kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/api.py", line 365, in _call_kw_model
    result = method(recs, *args, **kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/models.py", line 1527, in load_views
    result['fields_views'] = {
  File "/opt/odoo/custom/src/odoo/odoo/models.py", line 1528, in <dictcomp>
    v_type: self.fields_view_get(v_id, v_type if v_type != 'list' else 'tree',
  File "/opt/odoo/auto/addons/web/models/models.py", line 220, in fields_view_get
    r = super().fields_view_get(view_id, view_type, toolbar, submenu)
  File "/opt/odoo/custom/src/odoo/odoo/models.py", line 1618, in fields_view_get
    xarch, xfields = view.postprocess_and_fields(etree.fromstring(result['arch']), model=self._name)
  File "/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_ui_view.py", line 851, in postprocess_and_fields
    arch, name_manager = self._postprocess_view(node, model, validate=validate)
  File "/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_ui_view.py", line 863, in _postprocess_view
    self.postprocess(node, [], editable, name_manager)
  File "/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_ui_view.py", line 963, in postprocess
    self.postprocess(child, current_node_path, node_info['editable'], name_manager)
  File "/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_ui_view.py", line 958, in postprocess
    transfer_node_to_modifiers(node, node_info['modifiers'], self._context, current_node_path)
  File "/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_ui_view.py", line 66, in transfer_node_to_modifiers
    modifiers.update(ast.literal_eval(attrs))
  File "/usr/local/lib/python3.8/ast.py", line 99, in literal_eval
    return _convert(node_or_string)
  File "/usr/local/lib/python3.8/ast.py", line 88, in _convert
    return dict(zip(map(_convert, node.keys),
  File "/usr/local/lib/python3.8/ast.py", line 98, in _convert
    return _convert_signed_num(node)
  File "/usr/local/lib/python3.8/ast.py", line 75, in _convert_signed_num
    return _convert_num(node)
  File "/usr/local/lib/python3.8/ast.py", line 66, in _convert_num
    _raise_malformed_node(node)
  File "/usr/local/lib/python3.8/ast.py", line 63, in _raise_malformed_node
    raise ValueError(f'malformed node or string: {node!r}')
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 639, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 315, in _handle_exception
    raise exception.with_traceback(None) from new_cause
ValueError: malformed node or string: <_ast.Call object at 0x7f32a2c8a520>
```

Everything works with it as is. The truth is I don't recall why I needed this change :thinking: but it seems necessary, right? Or am I doing anything wrong somewhere else?

Reply:
The question is that if your expressing is working to show conditionally things. If you quote the whole expression as a string, I doubt it works as intended.

Reply:
Yes, you are right. I already added the correct fix to make it work correctly :+1: